src/layouts/MainLayout.vue

```diff
@@ -182,7 +182,8 @@ export default {
     },
   },
   async mounted() {
-    await this.autoLogin(this.$route.query.returnUrl);
+    console.log("MainLayout.mounted()");
```

Comment:
remove console log

Reply:
[removing console log and commented code](https://github.com/telosnetwork/telos-wallet/pull/89/commits/c5ae1c373048f990013ce7cdfde1882dd395fd5b)